GoTeamApp/GoTeamApp/AppDelegate.swift

```diff
@@ -26,32 +26,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
           configuration.server = "https://social-location-reminder.herokuapp.com/parse"
         })
       )
-      
-      let username = "test1@gmail.com"
-      let password = "twk4672kw"
-      
-      //    Parse User Sign Up
-      //    let user = PFUser()
-      //    let email = "test1@gmail.com"
-      //    user.username = username
-      //    user.password = password
-      //    user.email = email
-      //    user.signUpInBackground { (succees: Bool, error: Error?) in
-      //      if let error = error {
-      //        print("Parse Sign Up Error: \(error.localizedDescription)")
-      //      } else {
-      //        print("Sign up succeeded")
-      //      }
-      //    }
-      
-      // Parse User Login
-      PFUser.logInWithUsername(inBackground: username, password: password) { (user: PFUser?, error: Error?) in
-        if user != nil {
-          print("Sucessful login to Parse Server")
-        } else {
-          print("Parse Login Error \(error!.localizedDescription)")
-        }
-      }
```

Comment:
Yes, I tested it and Parse still worked. This was a test login to Parse which I added at first to test the connectivity. Later you created DataStoreServices which use different login credentials. I assume the test login in AppDelegate is no longer needed. Sorry for making the edit without proper comments.